src/add-to-registry.js

```diff
@@ -36,20 +35,27 @@ const YAML = require('yaml');
             "services": [].concat(installYmlData.services).flat(),
             "adobeRecommended": adobeRecommended,
             "keywords": [].concat(packageJsonData.keywords),
+            "status": TEMPLATE_STATUS_APPROVED,
```

Comment:
Please could you take a look on:
https://github.com/adobe/aio-template-submission/blob/d0018d5f8cdb0b5230338580e91154a4bb658779/src/add-to-registry.js#L54

Is it correc tlogic that if template "isInRegistry(templateData.name) === false" that it will be added with Approved status (but not like verification is in progress)?